src/lib/layouts/workflow-header.svelte

```diff
@@ -122,6 +136,27 @@
           class="overflow-hidden text-ellipsis"
         />
       </h1>
+      <div class="flex gap-4 items-center">
```

Comment:
A little responsive enhancement ⬇️ 

```suggestion
      <div class="flex gap-4 items-center flex-wrap">
```

Reply:
Nice